cloud_blobstore/s3.py

```diff
@@ -149,11 +149,18 @@ def upload_file_handle(
             self,
             bucket: str,
             key: str,
-            src_file_handle: typing.BinaryIO):
+            src_file_handle: typing.BinaryIO,
+            content_type: typing.Optional[str] = None,
+            metadata: typing.Optional[dict] = None):
+        if content_type or metadata:
+            extra_args = {}
+            extra_args.update({'ContentType': content_type} if content_type else {})
+            extra_args.update({'Metadate': content_type} if content_type else {})
         self.s3_client.upload_fileobj(
             src_file_handle,
             Bucket=bucket,
             Key=key,
+            ExtraArgs=extra_args
```

Comment:
If neither content_type nor metadata are specified, then extra_args is undefined.

Reply:
Fixed